deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/LaunchEventCollector.java

```diff
@@ -218,6 +233,18 @@ public void collectEvents() throws Exception {
         }
     }
 
+    private void sendEventEmail(String eventType, HostBean hostBean, GroupBean groupBean) {
+        LOG.info("Entered");
+        if (groupBean.getEmail_recipients() != null || groupBean.getPager_recipients() != null) {// what is watch recipients?? 
+            String recipients = groupBean.getEmail_recipients();
+            // String webLink = deployBoardUrlPrefix + String.format("/env/%s/%s/host/%s",  , ,hostBean.hostName);
+            String subject = String.format("Host %s Notification", eventType);
```

Comment:
Can you put something like this:

String.format("Autoscaling - Group <%s>  host %s notification,  ", groupName, eventType)